### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by jonsmithers**

### CODE REVIEW

File: webclient/src/services/todo-service.js [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
     constructor(host) {
         this.host = host + '/todo/';
     }
 
-    getAll() {
-        return fetch(this.host)
-            .then(response => response.json());
+    async getAll() {
+        var response = await fetch(this.host);
+        return response.json();
```
Comment (jonsmithers) [inline webclient/src/services/todo-service.js]: `.json()` actually returns a promise, which Is surprising to me. We'd technically have to add another await here. 

So:

```javascript
var response = await fetch(this.host);
return await response.json();
```

We could just gloss over this if we use the `.then()` syntax. I think I'll revert back to that.

Comment (billy-hardy) [reply webclient/src/services/todo-service.js]: I think when interfacing with async functions, a promise is returned to the calling function, so the await wouldn't really be necessary

Comment (jonsmithers) [reply webclient/src/services/todo-service.js]: You are quite right

File: install_yarn.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# this script will install yarn according to https://yarnpkg.com/lang/en/docs/install/#linux-tab
+
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update && sudo apt-get install yarn
```
Comment (jonsmithers) [inline install_yarn.sh]: The VM that I sent does not have node_modules installed, and it seems they can only be installed with `yarn`. I will have to make the students install yarn and then install npm dependencies. Unless I can change this when I meet Marc on Monday.

Comment (jonsmithers) [reply install_yarn.sh]: (alternatively.....we could commit node_modules)

**Pull request merged**